index.html

```diff
@@ -421,12 +421,48 @@
 				<h3>Default Reading Order</h3>
 
 				<p>The default reading order MUST include at least one <a>primary resource</a>.</p>
+				
 			</section>
 
 			<section id="table-of-contents">
 				<h3>Table of Contents</h3>
 
-				<div class="ednote">Placeholder for identifying table of contents - whether embedded or by reference.</div>
+
+				<p>The table of contents is recommended in the infoset.</p>
+				
+				<p>
+					An author MAY omit the table of contents from the <a href="#manifest">manifest</a> if it is included in an HTML <code>nav</code> element identified by the role <code>doc-toc</code>&nbsp;[[!dpub-aria-1.0]]. 
```

Comment:
I agree with @HadrienGardeur. I am not enamored with `index.html` either, but the alternative is to get into some other mechanisms (akin the Apache settings that leads to `index.html`) to identify which one is the TOC. We could call it `nav.html` like in EPUB...:-) but `index.html` is the de-facto standard on the Web...